Allow area, device, and entity selectors to optionally support multiple selections like target selector (#20961)
home-assistant/home-assistant.github.io: source/_docs/blueprint/selectors.markdown

## Area selector

> The area selector shows an area finder that can pick a single or multiple areas base on the selector configuration. The value of the input will be the area ID, or a list of area IDs, based on the user selection.

Reviewer: ```suggestion
The area selector shows an area finder that can pick a single or multiple areas based on the selector configuration. The value of the input will be the area ID, or a list of area IDs, based on the user selection.
```

> The area selector shows an area finder that can pick a single or multiple areas based on the selector configuration. The value of the input will be the area ID, or a list of area IDs, based on the user selection.

Reviewer 2: ```suggestion
The area selector shows an area finder that can pick a single or multiple areas based on the selector configuration. The value of the input will be the area ID, or a list of area IDs, based on if `multiple` is set to true.
```

## Device selector

> The device selector shows a device finder that can pick a single or multiple devices based on the selector configuration. The value of the input will contain the device ID or a list of device IDs, based on the user selection.

Reviewer 2: ```suggestion
The device selector shows a device finder that can pick a single or multiple devices based on the selector configuration. The value of the input will contain the device ID or a list of device IDs, based on if `multiple` is set to true.
```

## Entity selector

The entity selector shows an entity finder that can pick a single entity or a list of entities based on the selector configuration.

> The value of the input will contain the entity ID, or list of entity IDs, based on the user selection.

Reviewer 2: ```suggestion
The value of the input will contain the entity ID, or list of entity IDs, based on if `multiple` is set to true.
```
